[8.x] Upgrade EUI to v98.2.1-borealis.1

mgadewoll opened the pull request:
>[!IMPORTANT]
This PR introduces the Borealis enabled EUI package to `8.x`.
The intention is to support new features being developed that need to live in `main` as well as `8.x`.


>[!IMPORTANT]
All `8.x` versions should continue to use the current Amsterdam theme. To ensure this even with upcoming changes to the default theme for EUI, we're adding `euiThemeAmsterdam` on `EuiProvider` usages manually.

## Description

This PR introduces all previous Borealis related PR changes but excludes the specific changes to support `@elastic/eui-theme-borealis` as only Amsterdam is supported as theme in `8.x`.

Previous PRs

- https://github.com/elastic/kibana/pull/199993
- https://github.com/elastic/kibana/pull/201049
- https://github.com/elastic/kibana/pull/202073

---

`v97.3.1`⏩`v98.2.1-borealis.1`

_[Questions? Please see our Kibana upgrade FAQ.](https://github.com/elastic/eui/blob/main/wiki/eui-team-processes/upgrading-kibana.md#faq-for-kibana-teams)_

---

# `@elastic/eui`

## [`v98.2.1`](https://github.com/elastic/eui/releases/v98.2.1)

- Updated the EUI theme color values to use a full 6 char hex code format ([#8244](https://github.com/elastic/eui/pull/8244))

## [`v98.2.0`](https://github.com/elastic/eui/releases/v98.2.0)

- Added two new icons: `contrast` and `contrastHigh` ([#8216](https://github.com/elastic/eui/pull/8216))
- Updated `EuiDataGrid` content to have a transparent background. ([#8220](https://github.com/elastic/eui/pull/8220))

**Accessibility**

- When the tooltips components (`EuiTooltip`, `EuiIconTip`) are used inside components that handle the Escape key (like `EuiFlyout` or `EuiModal`), pressing the Escape key will now only close the tooltip and not the entire wrapping component. ([#8140](https://github.com/elastic/eui/pull/8140))
- Improved the accessibility of `EuiCodeBlock`s by ([#8195](https://github.com/elastic/eui/pull/8195))
  - adding screen reader only labels
  - adding `role="dialog"` on in fullscreen mode
  - ensuring focus is returned on closing fullscreen mode
  
# Borealis updates
  
  - [Visual Refresh] Update color token mappings ([#8211](https://github.com/elastic/eui/pull/8211))
  - [Visual Refresh] Introduce shared popover arrow styles to Borealis ([#8212](https://github.com/elastic/eui/pull/8212))
  - [Visual Refresh] Add forms.maxWidth token ([#8221](https://github.com/elastic/eui/pull/8221))
  - [Visual Refresh] Set darker shade for subdued text ([#8224](https://github.com/elastic/eui/pull/8224))
  - [Visual Refresh] Remove support for accentSecondary badges ([#8224](https://github.com/elastic/eui/pull/8227))
  - [Visual Refresh] Add severity vis colors ([#8247](https://github.com/elastic/eui/pull/8247))

---

# `@elastic/eui`

## [`v98.1.0`](https://github.com/elastic/eui/releases/v98.1.0)

- Updated `EuiBetaBadge` with a new `warning` color variant ([#8177](https://github.com/elastic/eui/pull/8177))

**Accessibility**

- Ensures `autoFocus` on `EuiSelectableList` triggers initial focus ([#8091](https://github.com/elastic/eui/pull/8091))
- Improved the accessibility of `EuiSearchBarFilters` by:  ([#8091](https://github.com/elastic/eui/pull/8091))
  - adding a more descriptive `aria-label` to selection filter buttons
  - ensuring the selection listbox is initially focused when opening a selection popover
- Improved the accessibility experience of tabs (EuiTab, EuiTabs): tab group is a tab stop and tabs can be traversed with arrow keys. ([#8116](https://github.com/elastic/eui/pull/8116))
- Updated `EuiCodeBlock` with a new `copyAriaLabel` prop, which allows setting a custom screen reader label on the copy button. ([#8176](https://github.com/elastic/eui/pull/8176))

**CSS-in-JS conversions**

- Removed the following global Sass variables: ([#8169](https://github.com/elastic/eui/pull/8169))
  - `$euiButtonMinWidth`
  - `$euiDatePickerCalendarWidth`
- Removed the following Sass animations: ([#8169](https://github.com/elastic/eui/pull/8169))
  - `euiAnimFadeIn`
  - `euiGrow`
  - `focusRingAnimate`
  - `focusRingAnimateLarge`
  - `euiButtonActive`
- Removed the following Sass mixins: ([#8169](https://github.com/elastic/eui/pull/8169))
  - `euiFullHeight`
  - `euiSlightShadowHover`
  - `datePickerArrow`

# Borealis updates

- [Visual Refresh] Fix euiColorFullShade Sass variable mapping (https://github.com/elastic/eui/pull/8178)

---

# `@elastic/eui`

## [`v98.0.0`](https://github.com/elastic/eui/releases/v98.0.0)

**Bug fixes**

-  Fixed an `EuiDataGrid` bug where column actions where not clickable when `EuiDataGrid` with `columnVisibility.canDragAndDropColumns` was used inside a modal ([#8135](https://github.com/elastic/eui/pull/8135))

**Breaking changes**

- Removed `EuiFormRow`'s deprecated `columnCompressedSwitch` display prop. Use `columnCompressed` instead ([#8113](https://github.com/elastic/eui/pull/8113))

# Borealis updates

**Bug fixes**

- Fixed computed border token mapping (https://github.com/elastic/eui/pull/8170)

elasticmachine commented:
<!-- buildkite-pr-bot-trigger-comment -->:robot: Jobs for this PR can be triggered through checkboxes. :construction: 

:information_source: To trigger the CI, please tick the checkbox below :point_down: 

- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-pull-request --> Click to trigger **kibana-pull-request** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-project-from-pr --> Click to trigger **kibana-deploy-project-from-pr** for this PR!

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/261928)
* Commit: 61209c5ec4a65081d842a9153776e3c5ad59478e

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/261928#0193da75-b054-4deb-bfaa-ad1705d2bace)

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/261928","number":261928,"commit":"61209c5ec4a65081d842a9153776e3c5ad59478e"}],"number":261928}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/261962)
* Commit: 7663f93cd7203f62d1d2a3ecdfef31242945b2b6

### Failed CI Steps
* [Jest Tests #7](https://buildkite.com/elastic/kibana-pull-request/builds/261962#0193dacf-75f9-4e43-9bb6-f4a4ead86a02)
* [Jest Tests #7](https://buildkite.com/elastic/kibana-pull-request/builds/261962#0193daf2-99e2-4f2f-be38-41040307f060)
* [Jest Tests #11](https://buildkite.com/elastic/kibana-pull-request/builds/261962#0193dacf-75fa-4895-85bb-1fdd7eb58976)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/261962#0193dacf-75f9-4e43-9bb6-f4a4ead86a02) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/261962/jobs/0193dacf-75f9-4e43-9bb6-f4a4ead86a02/artifacts/0193daf2-89f7-47ea-a1a8-d73a528e26f8) Jest Tests #<span></span>7 / ExceptionListHeader should render the List Header with name, default description and disabled actions because user can not edit details
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/261962#0193daf2-99e2-4f2f-be38-41040307f060) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/261962/jobs/0193daf2-99e2-4f2f-be38-41040307f060/artifacts/0193daf7-4dcc-4c41-8f44-05d5e7b6eb80) Jest Tests #<span></span>7 / ExceptionListHeader should render the List Header with name, default description and disabled actions because user can not edit details
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/261962#0193dacf-75f9-4e43-9bb6-f4a4ead86a02) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/261962/jobs/0193dacf-75f9-4e43-9bb6-f4a4ead86a02/artifacts/0193daf2-89f6-4e82-905c-27c18eb59ea5) Jest Tests #<span></span>7 / HeaderMenu should render button icon disabled
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/261962#0193daf2-99e2-4f2f-be38-41040307f060) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/261962/jobs/0193daf2-99e2-4f2f-be38-41040307f060/artifacts/0193daf7-4dcd-4062-9ba6-ce281c3993ea) Jest Tests #<span></span>7 / HeaderMenu should render button icon disabled
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/261962#0193dacf-75fa-4895-85bb-1fdd7eb58976) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/261962/jobs/0193dacf-75fa-4895-85bb-1fdd7eb58976/artifacts/0193daf2-5ad5-4f7b-9a53-3031291b1c39) Jest Tests #<span></span>11 / HeatmapStyleEditor is rendered
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/261962#0193dacf-75f9-4e43-9bb6-f4a4ead86a02) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/261962/jobs/0193dacf-75f9-4e43-9bb6-f4a4ead86a02/artifacts/0193daf2-89f5-48df-87bd-0d72107fdf95) Jest Tests #<span></span>7 / MenuItems should render all menu actions enabled
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/261962#0193daf2-99e2-4f2f-be38-41040307f060) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/261962/jobs/0193daf2-99e2-4f2f-be38-41040307f060/artifacts/0193daf7-4dcc-49c3-af6a-e2a6d5cc0573) Jest Tests #<span></span>7 / MenuItems should render all menu actions enabled
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/261962#0193dacf-75f9-4e43-9bb6-f4a4ead86a02) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/261962/jobs/0193dacf-75f9-4e43-9bb6-f4a4ead86a02/artifacts/0193daf2-89f5-47ee-bcb7-ba8645512158) Jest Tests #<span></span>7 / MenuItems should render delete action disabled when "canUserEditList" is "false"
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/261962#0193daf2-99e2-4f2f-be38-41040307f060) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/261962/jobs/0193daf2-99e2-4f2f-be38-41040307f060/artifacts/0193daf7-4dcd-43fb-8d34-f91941a96f4b) Jest Tests #<span></span>7 / MenuItems should render delete action disabled when "canUserEditList" is "false"

### History
* :broken_heart: [Build #261928](https://buildkite.com/elastic/kibana-pull-request/builds/261928) failed 61209c5ec4a65081d842a9153776e3c5ad59478e

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/261962","number":261962,"commit":"7663f93cd7203f62d1d2a3ecdfef31242945b2b6"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/261928","number":261928,"commit":"61209c5ec4a65081d842a9153776e3c5ad59478e"}],"number":261962}
buildkite-pr-comment-->

mgadewoll committed:
feat: introduce borealis eui package version 98.2.1-borealis.1

mgadewoll committed:
refactor: set static EuiThemeAmsterdam on EuiProvider usages

- this ensures that upcoming changes to the default theme on EUI side won't progagate unexpectedly to 8.x

mgadewoll committed:
fix: type errors

mgadewoll committed:
test: update tests and snapshots

elasticmachine commented:
## :yellow_heart: Build succeeded, but was flaky
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/262158)
* Commit: 1f955561772ca365ebc1bf345622da98fc5009cf

### Failed CI Steps
* [FTR Configs #11](https://buildkite.com/elastic/kibana-pull-request/builds/262158#0193de4f-143a-4f96-afb4-03fef10e96fa)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/262158#0193de4f-143a-4f96-afb4-03fef10e96fa) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/262158/jobs/0193de4f-143a-4f96-afb4-03fef10e96fa/artifacts/0193de72-172c-405b-a790-5ed6b7ceae33) FTR Configs #<span></span>11 / security APIs - Session Idle Session Idle cleanup should properly clean up session expired because of idle timeout when providers override global session config

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/2ac61869b529ef3fd728d1f1bd1f7f224106bca0) | [after](https://github.com/elastic/kibana/commit/1f955561772ca365ebc1bf345622da98fc5009cf) | diff |
| --- | --- | --- | --- |
| `controls` | 364 | 368 | +4 |
| `dashboard` | 688 | 693 | +5 |
| `infra` | 1531 | 1540 | +9 |
| `kubernetesSecurity` | 202 | 211 | +9 |
| `maps` | 1259 | 1264 | +5 |
| `ml` | 2129 | 2138 | +9 |
| `visTypeVega` | 280 | 289 | +9 |
| total |  |  | +50 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/2ac61869b529ef3fd728d1f1bd1f7f224106bca0) | [after](https://github.com/elastic/kibana/commit/1f955561772ca365ebc1bf345622da98fc5009cf) | diff |
| --- | --- | --- | --- |
| `aiops` | 621.6KB | 621.7KB | +104.0B |
| `alerting` | 98.6KB | 98.7KB | +26.0B |
| `apm` | 3.5MB | 3.5MB | +52.0B |
| `canvas` | 1.1MB | 1.1MB | +28.0B |
| `cloudDataMigration` | 6.1KB | 6.2KB | +26.0B |
| `cloudSecurityPosture` | 502.9KB | 502.9KB | +26.0B |
| `controls` | 460.0KB | 494.6KB | +34.7KB |
| `core` | 60.9KB | 60.9KB | +1.0B |
| `crossClusterReplication` | 146.5KB | 146.5KB | +26.0B |
| `dashboard` | 632.4KB | 667.9KB | +35.5KB |
| `dataQuality` | 28.5KB | 28.5KB | +26.0B |
| `dataUsage` | 243.0KB | 243.0KB | +26.0B |
| `dataViewManagement` | 140.0KB | 140.0KB | +26.0B |
| `dataVisualizer` | 614.8KB | 614.9KB | +130.0B |
| `devTools` | 8.0KB | 8.0KB | +26.0B |
| `discover` | 813.8KB | 813.8KB | +26.0B |
| `enterpriseSearch` | 2.6MB | 2.6MB | +26.0B |
| `entityManagerApp` | 12.5KB | 12.5KB | +26.0B |
| `eventAnnotationListing` | 231.7KB | 231.8KB | +26.0B |
| `exploratoryView` | 182.9KB | 183.0KB | +26.0B |
| `expressionPartitionVis` | 35.6KB | 35.6KB | -1.0B |
| `filesManagement` | 125.5KB | 125.5KB | +26.0B |
| `fileUpload` | 951.4KB | 951.4KB | -1.0B |
| `graph` | 418.0KB | 418.1KB | +26.0B |
| `grokdebugger` | 11.2KB | 11.2KB | +26.0B |
| `home` | 152.2KB | 152.2KB | +26.0B |
| `imageEmbeddable` | 69.5KB | 69.6KB | +52.0B |
| `indexLifecycleManagement` | 166.1KB | 166.1KB | +26.0B |
| `infra` | 1.7MB | 1.8MB | +42.8KB |
| `ingestPipelines` | 406.7KB | 406.7KB | +1.0B |
| `inputControlVis` | 52.3KB | 52.3KB | +26.0B |
| `inventory` | 234.1KB | 234.1KB | +26.0B |
| `investigateApp` | 484.3KB | 484.3KB | +26.0B |
| `kibanaOverview` | 54.9KB | 54.9KB | +26.0B |
| `kibanaReact` | 197.1KB | 197.1KB | +1.0B |
| `kibanaUtils` | 60.8KB | 60.8KB | +1.0B |
| `kubernetesSecurity` | 216.4KB | 259.7KB | +43.3KB |
| `lens` | 1.5MB | 1.5MB | +26.0B |
| `licenseManagement` | 46.8KB | 46.8KB | +26.0B |
| `links` | 53.0KB | 53.1KB | +52.0B |
| `logstash` | 33.6KB | 33.7KB | +26.0B |
| `management` | 48.5KB | 48.6KB | +26.0B |
| `maps` | 3.0MB | 3.0MB | +35.5KB |
| `metricsDataAccess` | 99.7KB | 99.7KB | +26.0B |
| `ml` | 4.7MB | 4.7MB | +43.0KB |
| `monitoring` | 533.6KB | 533.6KB | +26.0B |
| `observabilityAIAssistantApp` | 295.1KB | 295.1KB | +19.0B |
| `observabilityLogsExplorer` | 147.7KB | 147.7KB | +26.0B |
| `observabilityOnboarding` | 283.1KB | 283.1KB | +26.0B |
| `painlessLab` | 18.7KB | 18.8KB | +26.0B |
| `profiling` | 406.3KB | 406.3KB | +26.0B |
| `remoteClusters` | 79.4KB | 79.5KB | +26.0B |
| `rollup` | 115.8KB | 115.8KB | +26.0B |
| `savedObjectsManagement` | 86.9KB | 87.0KB | +26.0B |
| `searchAssistant` | 165.0KB | 165.0KB | +26.0B |
| `searchHomepage` | 55.0KB | 55.1KB | +26.0B |
| `searchIndices` | 167.7KB | 167.7KB | +26.0B |
| `searchInferenceEndpoints` | 98.8KB | 98.8KB | +26.0B |
| `searchNotebooks` | 14.2KB | 14.2KB | +26.0B |
| `searchPlayground` | 163.2KB | 163.2KB | +26.0B |
| `searchprofiler` | 49.0KB | 49.1KB | +26.0B |
| `security` | 540.7KB | 540.7KB | +1.0B |
| `securitySolution` | 18.5MB | 18.5MB | +160.0B |
| `serverlessSearch` | 339.4KB | 339.5KB | +52.0B |
| `slo` | 843.1KB | 843.3KB | +130.0B |
| `snapshotRestore` | 263.3KB | 263.3KB | +26.0B |
| `streamsApp` | 116.4KB | 116.5KB | +26.0B |
| `synthetics` | 1.1MB | 1.1MB | +52.0B |
| `timelines` | 33.0KB | 33.0KB | +26.0B |
| `transform` | 475.5KB | 475.5KB | +26.0B |
| `triggersActionsUi` | 1.7MB | 1.7MB | +286.0B |
| `uptime` | 468.4KB | 468.4KB | +26.0B |
| `ux` | 168.2KB | 168.3KB | +26.0B |
| `visTypeTimeseries` | 507.3KB | 507.3KB | +1.0B |
| `visTypeVega` | 1.9MB | 1.9MB | +42.8KB |
| `watcher` | 166.5KB | 166.5KB | +26.0B |
| total |  |  | +279.8KB |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/2ac61869b529ef3fd728d1f1bd1f7f224106bca0) | [after](https://github.com/elastic/kibana/commit/1f955561772ca365ebc1bf345622da98fc5009cf) | diff |
| --- | --- | --- | --- |
| `advancedSettings` | 7.7KB | 7.7KB | +26.0B |
| `banners` | 10.7KB | 10.7KB | +26.0B |
| `cases` | 159.6KB | 159.6KB | +26.0B |
| `cloudLinks` | 30.4KB | 30.5KB | +26.0B |
| `console` | 38.6KB | 38.6KB | +26.0B |
| `core` | 451.0KB | 451.4KB | +447.0B |
| `dashboard` | 51.6KB | 51.6KB | +26.0B |
| `dashboardEnhanced` | 18.1KB | 18.1KB | +26.0B |
| `data` | 420.0KB | 420.1KB | +26.0B |
| `dataViewEditor` | 13.8KB | 13.8KB | +26.0B |
| `dataViewFieldEditor` | 27.0KB | 27.0KB | +26.0B |
| `embeddable` | 49.0KB | 49.0KB | +26.0B |
| `expressionError` | 7.5KB | 7.5KB | +26.0B |
| `expressionGauge` | 16.5KB | 16.5KB | +26.0B |
| `expressionHeatmap` | 17.4KB | 17.4KB | +26.0B |
| `expressionImage` | 6.8KB | 6.9KB | +26.0B |
| `expressionLegacyMetricVis` | 12.7KB | 12.7KB | +26.0B |
| `expressionMetric` | 8.1KB | 8.2KB | +26.0B |
| `expressionMetricVis` | 17.1KB | 17.1KB | +26.0B |
| `expressionPartitionVis` | 29.3KB | 29.4KB | +27.0B |
| `expressionRepeatImage` | 8.6KB | 8.6KB | +26.0B |
| `expressionRevealImage` | 8.6KB | 8.6KB | +26.0B |
| `expressionShape` | 23.1KB | 23.1KB | +26.0B |
| `expressionTagcloud` | 13.5KB | 13.5KB | +26.0B |
| `expressionXY` | 42.2KB | 42.2KB | +26.0B |
| `fleet` | 173.2KB | 173.2KB | +26.0B |
| `globalSearchBar` | 30.0KB | 30.0KB | +26.0B |
| `guidedOnboarding` | 29.8KB | 29.8KB | +26.0B |
| `indexManagement` | 46.9KB | 46.9KB | +26.0B |
| `ingestPipelines` | 17.7KB | 17.7KB | +26.0B |
| `inspector` | 25.1KB | 25.2KB | +26.0B |
| `interactiveSetup` | 60.0KB | 60.0KB | +26.0B |
| `kbnUiSharedDeps-npmDll` | 5.8MB | 5.9MB | +63.9KB |
| `kibanaReact` | 40.0KB | 40.0KB | +26.0B |
| `kibanaUtils` | 68.8KB | 68.9KB | +27.0B |
| `licensing` | 11.9KB | 11.9KB | +26.0B |
| `maps` | 54.5KB | 54.5KB | +26.0B |
| `mockIdpPlugin` | 10.9KB | 10.9KB | +26.0B |
| `newsfeed` | 13.0KB | 13.0KB | +26.0B |
| `observability` | 105.1KB | 105.1KB | +26.0B |
| `observabilityAIAssistantApp` | 16.6KB | 16.6KB | +26.0B |
| `osquery` | 47.2KB | 47.2KB | +26.0B |
| `presentationPanel` | 43.5KB | 43.5KB | +26.0B |
| `reporting` | 52.9KB | 53.0KB | +26.0B |
| `runtimeFields` | 14.1KB | 14.1KB | +26.0B |
| `savedObjects` | 17.8KB | 17.9KB | +26.0B |
| `savedObjectsTagging` | 22.8KB | 22.9KB | +26.0B |
| `security` | 68.1KB | 68.1KB | +26.0B |
| `serverless` | 14.3KB | 14.4KB | +26.0B |
| `share` | 58.0KB | 58.0KB | +26.0B |
| `spaces` | 34.2KB | 34.3KB | +26.0B |
| `telemetry` | 25.1KB | 25.1KB | +26.0B |
| `uiActions` | 24.2KB | 24.2KB | +26.0B |
| `unifiedSearch` | 38.4KB | 38.4KB | +26.0B |
| `upgradeAssistant` | 24.8KB | 24.8KB | +26.0B |
| `visDefaultEditor` | 24.2KB | 24.2KB | +26.0B |
| `visTypeMarkdown` | 9.9KB | 9.9KB | +26.0B |
| `visTypeTable` | 19.4KB | 19.4KB | +26.0B |
| `visTypeTimelion` | 13.7KB | 13.7KB | +26.0B |
| `visTypeTimeseries` | 20.6KB | 20.7KB | +26.0B |
| `visTypeVega` | 36.3KB | 36.3KB | +26.0B |
| `visTypeVislib` | 15.3KB | 15.3KB | +26.0B |
| `visualizations` | 64.9KB | 65.0KB | +26.0B |
| total |  |  | +65.8KB |



### History
* :broken_heart: [Build #261962](https://buildkite.com/elastic/kibana-pull-request/builds/261962) failed 7663f93cd7203f62d1d2a3ecdfef31242945b2b6
* :broken_heart: [Build #261928](https://buildkite.com/elastic/kibana-pull-request/builds/261928) failed 61209c5ec4a65081d842a9153776e3c5ad59478e

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/262158","number":262158,"commit":"1f955561772ca365ebc1bf345622da98fc5009cf"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/261962","number":261962,"commit":"7663f93cd7203f62d1d2a3ecdfef31242945b2b6"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/261928","number":261928,"commit":"61209c5ec4a65081d842a9153776e3c5ad59478e"}],"number":262158}
buildkite-pr-comment-->

elasticmachine commented:
Pinging @elastic/eui-team (EUI)

elasticmachine commented:
<!-- buildkite-pr-bot-trigger-comment -->:robot: Jobs for this PR can be triggered through checkboxes. :construction: 

:information_source: To trigger the CI, please tick the checkbox below :point_down: 

- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-pull-request --> Click to trigger **kibana-pull-request** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-project-from-pr --> Click to trigger **kibana-deploy-project-from-pr** for this PR!

fkanout reviewed (approved):
Code review only (package.json) LGTM

elasticmachine commented:
Pinging @elastic/obs-ux-infra_services-team (Team:obs-ux-infra_services)

consulthys reviewed (approved):
LGT Stack Monitoring
(single color change)

jughosta commented on examples/data_view_field_editor_example/public/app.tsx:
```diff
@@ -136,7 +137,7 @@ const DataViewFieldEditorExample = ({ dataView, dataViewFieldEditor }: Props) =>
 
   return (
-    <EuiProvider>
+    <EuiProvider theme={EuiThemeAmsterdam}>
```
Looks like this change forces the dark mode, does not it?

<img width="1420" alt="Screenshot 2024-12-19 at 14 50 44" src="https://github.com/user-attachments/assets/988c426b-24f2-4bdf-a697-fb70aeaef52f" />

mgadewoll replied:
@jughosta @stratoula 
I looked into it and as far as I can tell, this issue is not introduced by this PR.
The reason the examples are shown in Dark mode for you, is 
a) you have dark mode enables as OS settings
b) all examples use a separate `EuiProvider` which does not check on what color mode Kibana sets

EUI changed the `EuiProvider` a while back (version `95.12.0` ([release](https://github.com/elastic/eui/releases/tag/v95.12.0))) which enabled the `colorMode` to respect OS level settings _if_ no `colorMode` is set on the `EuiProvider` to override it. Previously the default `colorMode` was just `light`.

That means two things:
- this issue existed prior to this PR (and even on `main` I'm assuming) and would require all examples to either set a fixed `colorMode` or check on Kibanas colorMode to be in sync when updating color mode via the Kibana user settings.
- this issue even existed prio to the EUI change to respect OS level settings, just in the opposite direction: Examples would always load in light mode, but if Kibana is set to dark mode (user settings) then Kibana is dark but the examples still white

![Screenshot 2024-12-19 at 16 11 17](https://github.com/user-attachments/assets/4aec194a-9a27-4884-aaa0-5fd713832903)

If you agree, I'd say this is something that can/should be address as a separate change as it's not related to this PR or EUI specifically.

jughosta replied:
Sure, thanks!

Dosant replied:
Created https://github.com/elastic/kibana/issues/205033

jughosta commented on examples/field_formats_example/public/app.tsx:
```diff
@@ -168,7 +169,7 @@ const CreatingCustomFieldFormatEditor: React.FC<{ deps: Deps }> = (props) => {
 export const App: React.FC<{ deps: Deps }> = (props) => {
   return (
-    <EuiProvider>
+    <EuiProvider theme={EuiThemeAmsterdam}>
```
Same here
<img width="1167" alt="Screenshot 2024-12-19 at 14 55 13" src="https://github.com/user-attachments/assets/f43afe42-fab5-4d00-9fa7-60cdf698d206" />

JasonStoltz replied:
Thanks @jughosta, we're taking a look now to see what's going on.

MiriamAparicio reviewed (approved):
Changes for `obs-ux-infra_services-team` LGTM

stratoula commented on examples/esql_ast_inspector/public/app.tsx:
For some reason the example  app looks like it is on the dark mode, while it is not

<img width="1184" alt="image" src="https://github.com/user-attachments/assets/e1f75bad-520d-467d-b0d4-078703a458bf" />

JasonStoltz replied:
Looking now!

mgadewoll replied:
Cross linking to the conversation on [this](https://github.com/elastic/kibana/pull/204783#discussion_r1892475253)

rylnd reviewed (approved):
There were no changes to @elastic/security-detection-engine here, but I checked out this branch to verify that the workflows for our our areas continued to work/look correct on the Amsterdam theme.

SoniaSanzV reviewed (approved):
Kibana Management changes (`extend_index_management` snapshot) LGTM

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/264251)
* Commit: f0951765c34355ebba7026d2c3e26fd1ff521034
* Kibana Serverless Image: `docker.elastic.co/kibana-ci/kibana-serverless:pr-204783-f0951765c343`

### History
* :yellow_heart: [Build #262158](https://buildkite.com/elastic/kibana-pull-request/builds/262158) was flaky 1f955561772ca365ebc1bf345622da98fc5009cf
* :broken_heart: [Build #261962](https://buildkite.com/elastic/kibana-pull-request/builds/261962) failed 7663f93cd7203f62d1d2a3ecdfef31242945b2b6
* :broken_heart: [Build #261928](https://buildkite.com/elastic/kibana-pull-request/builds/261928) failed 61209c5ec4a65081d842a9153776e3c5ad59478e

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/264251","number":264251,"commit":"f0951765c34355ebba7026d2c3e26fd1ff521034"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/262158","number":262158,"commit":"1f955561772ca365ebc1bf345622da98fc5009cf"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/261962","number":261962,"commit":"7663f93cd7203f62d1d2a3ecdfef31242945b2b6"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/261928","number":261928,"commit":"61209c5ec4a65081d842a9153776e3c5ad59478e"}],"number":264251}
buildkite-pr-comment-->

mgadewoll commented:
@elastic/security-defend-workflows Would you be able to have a look too?

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/264365)
* Commit: b3952a1e874d7005cf1905f1b5d8f64435ed3ed8
* Kibana Serverless Image: `docker.elastic.co/kibana-ci/kibana-serverless:pr-204783-b3952a1e874d`

### History
* :green_heart: [Build #264251](https://buildkite.com/elastic/kibana-pull-request/builds/264251) succeeded f0951765c34355ebba7026d2c3e26fd1ff521034
* :yellow_heart: [Build #262158](https://buildkite.com/elastic/kibana-pull-request/builds/262158) was flaky 1f955561772ca365ebc1bf345622da98fc5009cf
* :broken_heart: [Build #261962](https://buildkite.com/elastic/kibana-pull-request/builds/261962) failed 7663f93cd7203f62d1d2a3ecdfef31242945b2b6
* :broken_heart: [Build #261928](https://buildkite.com/elastic/kibana-pull-request/builds/261928) failed 61209c5ec4a65081d842a9153776e3c5ad59478e

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/264365","number":264365,"commit":"b3952a1e874d7005cf1905f1b5d8f64435ed3ed8"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/264251","number":264251,"commit":"f0951765c34355ebba7026d2c3e26fd1ff521034"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/262158","number":262158,"commit":"1f955561772ca365ebc1bf345622da98fc5009cf"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/261962","number":261962,"commit":"7663f93cd7203f62d1d2a3ecdfef31242945b2b6"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/261928","number":261928,"commit":"61209c5ec4a65081d842a9153776e3c5ad59478e"}],"number":264365}
buildkite-pr-comment-->

eokoneyo reviewed (approved):
Shared UX changes LGTM
